desktop/src/scenes/Analyze/crew.jsx

```diff
@@ -0,0 +1,158 @@
+import CrewMemberShiftCard from '~/components/crewCards/crewMemberShiftCard'
+import Progress from '~/components/helpers/Progress'
+
+import {
+  foremanActions,
+  employeeActions,
+  projectActions,
+  taskActions,
+  shiftActions,
+} from '~/store/actions'
+import { crewActions } from '~/store/actions'
+import { crewSelectors, employeeSelectors } from '~/store/selectors'
+
+import moment from 'moment'
+
+export class Crew extends Component {
+  constructor(props) {
+    super(props)
+    const yesterday = new Date()
+    yesterday.setDate(yesterday.getDate() - 1)
+    this.state = {
+      day: moment.utc(yesterday).local().format('YYYY-MM-DD'),
+      isLoading: true,
+    }
+    this.setDay = this.setDay.bind(this)
+  }
+
+  componentDidMount = async () => {
+    Promise.all([
+      this.props.getAllEmployees(),
+      this.props.getAllProjects(),
+      this.props.getAllTasks(),
+      this.props.getShiftsInRange(
+        moment().subtract(365, `days`).format(`MM-DD-YY HH:mm:ss`),
+        moment().add(14, `days`).format(`MM-DD-YY HH:mm:ss`),
+      ),
+    ]).then(() => {
+      this.setState({ ...this.state, isLoading: false })
+    })
+  }
+
+  setDay = (day) => {
+    this.setState({ day })
+  }
+
+  componentDidUpdate = (prevProps) => {
+    if (prevProps.currentEmployee !== this.props.currentEmployee) {
+      var employeecrew = this.props.crews.find(
+        (crew) => crew.id === this.props.currentEmployee.crewId,
+      )
+      this.state.selectedCrew = {
+        label: employeecrew.name,
+        value: employeecrew.name,
+        id: employeecrew.id,
+        data: {},
+        name: employeecrew.name,
+      }
+    }
+  }
+
+  back = () => {
+    window.history.back()
+  }
+  render() {
+    if (this.state.isLoading)
+      return (
+        <div>
+          <Progress variant="circular" fullWidth fullHeight />
+        </div>
+      )
+    const { crews } = this.props
+    return (
+      <div className="w-screen h-full flex flex-col space-y-4 md:space-y-0">
+        <AppBar position="static" elevation={0}>
+          <Toolbar disableGutters className="min-h-0 mx-4 space-x-4">
+            <Day value={this.state.day} onChange={this.setDay} />
+            <Select
+              value={this.state.selectedCrew}
+              onChange={(selectedCrew) => {
+                this.props.updateFilter(selectedCrew.id)
+                console.log({ selectedCrew })
+                return this.setState({ selectedCrew })
+              }}
+              options={crews.map((item) => {
+                return {
+                  label: item.name,
+                  value: item.name,
+                  id: item.id,
+                  data: { ...item },
+                  name: item.name,
+                }
+              })}
+              className="text-slate-900 py-4 flex-grow"
+            />
+            <Tooltip title="Go Back" placement="bottom">
+              <IconButton
+                className="!hidden md:!block"
```

Comment:
Kinda weird that you have to make these important. Are they both just being overwritten by material UI?